src/Uno.UI/UI/Xaml/Controls/Control/Control.cs

```diff
@@ -109,8 +111,33 @@ private protected virtual void ChangeVisualState(bool useTransitions)
 
 		public event DependencyPropertyChangedEventHandler IsEnabledChanged;
 
-		[GeneratedDependencyProperty(DefaultValue = true, ChangedCallback = true, CoerceCallback = true, Options = FrameworkPropertyMetadataOptions.Inherits)]
-		public static DependencyProperty IsEnabledProperty { get; } = CreateIsEnabledProperty();
+		public static DependencyProperty IsEnabledProperty { get; } = 
+			DependencyProperty.Register(name: "IsEnabled", propertyType: typeof(bool), ownerType: typeof(Control),
+				typeMetadata: new FrameworkPropertyMetadata(
+					defaultValue: true
+					, options: FrameworkPropertyMetadataOptions.Inherits | FrameworkPropertyMetadataOptions.KeepCoercedWhenEquals
+					, backingFieldUpdateCallback: OnIsEnabledBackingFieldUpdate
+					, coerceValueCallback: (instance, baseValue) => ((Control)instance).CoerceIsEnabled(baseValue)
+					, propertyChangedCallback: (instance, args) => ((Control)instance).OnIsEnabledChanged(args)
+				));
+		
+		private static void OnIsEnabledBackingFieldUpdate(object instance, object newValue)
+		{
+			var typedInstance = instance as Control;
+			typedInstance!._IsEnabledPropertyBackingField = (bool)newValue;
+			typedInstance!._IsEnabledPropertyBackingFieldSet = true;
+		}
+		
+		private bool GetIsEnabledValue()
+		{
+			if (!_IsEnabledPropertyBackingFieldSet)
+			{
+				_IsEnabledPropertyBackingField = (bool)GetValue(IsEnabledProperty);
+				_IsEnabledPropertyBackingFieldSet = true;
+			}
+			return _IsEnabledPropertyBackingField;
+		}
+		private void SetIsEnabledValue(bool value) => SetValue(IsEnabledProperty, value);
```

Comment:
As it's now a flag on the enum, we no longer have to disable the generation for that property
```suggestion
		[GeneratedDependencyProperty(DefaultValue = true, ChangedCallback = true, CoerceCallback = true, Options = FrameworkPropertyMetadataOptions.Inherits | FrameworkPropertyMetadataOptions.KeepCoercedWhenEquals)]
		public static DependencyProperty IsEnabledProperty { get; } = CreateIsEnabledProperty();
```